src/test/java/core/basesyntax/OperationHandlerTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.handler.DecreaseOperationHandler;
+import core.basesyntax.handler.IncreaseOperationHandler;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationHandlerTest {
+    private static final OperationHandler operationHandlerDecrease = new DecreaseOperationHandler();
```

Comment:
let's initialise our objects in `BeforeClass`

Reply:
done